### PR TITLE
Java directions; data permissions; new GPK

### PR DESCRIPTION
**Pull request by garys-esri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,9 +57,10 @@ Hardware requirements for this template are the same as those for ArcGIS Runtime
 This section is for developers who just need to quickly build and run the application.
  
 ### Verify your development environment
-* Ensure the JavaSE SDK is installed
-    * Java is installed and added to your path and the environment variable `JAVA_HOME` to be set to this location
-    * To verify your Java Installation: Open Command Prompt> `java -version` and verify it runs and returns the version correctly 
+* Ensure the Java Development Kit (JDK) is installed
```
Comment (csmoore) [inline README.md]: Opps, I guess we never noticed this before - good catch

File: license-ThirdParty.txt [hunk 2]
```diff
@@ -1,8 +1,24 @@
...
+
+This application uses the following libraries whose use is governed by the GNU Lesser General Public License (LGPL):
 
 Beans Binding (JSR-295)
 
+See below for the text of the LGPL.
+
+*****
+
+SRTM dataset citation:
```
Comment (csmoore) [inline license-ThirdParty.txt]: Very thorough

**Comment by csmoore** [comment]
Looks good

**Comment by garys-esri** [comment]
@csmoore, thanks for the help!

**Pull request merged**